boost-maven-plugin/src/main/java/boost/project/LibertyBoostMojo.java

```diff
@@ -292,21 +312,22 @@ private void installMissingFeatures() throws MojoExecutionException {
 
         executeMojo(getPlugin(), goal("install-feature"), configuration(element(name("serverName"), libertyServerName),
                 element(name("features"), element(name("acceptLicense"), "true")
-
-                )), getExecutionEnvironment());
+        )), getExecutionEnvironment());
     }
 
     /**
      * Invoke the liberty-maven-plugin to run the package-server goal
      *
      */
     private void createUberJar() throws MojoExecutionException {
-
         // Package server into runnable jar
         executeMojo(getPlugin(),
-                goal("package-server"), configuration(element(name("isInstall"), "false"),
-                        element(name("include"), "minify,runnable"), element(name("serverName"), libertyServerName)),
-                getExecutionEnvironment());
+            goal("package-server"), configuration(element(name("isInstall"), "false"),
+                    element(name("include"), "minify,runnable"),
+                    element(name("attach"), "true"),
```

Comment:
Do we still want "attach" specified here since we are not changing the name of the deployed archive?

Reply:
Yes, otherwise the project artifact will be the thinned artifact (it is changed to that in the `install-apps` goal).